src/installer/pkg/projects/Directory.Build.props

```diff
@@ -100,7 +100,7 @@
   <Import Project="$(RIDPropsFile)" />
 
   <ItemGroup>
-    <_buildingOnRID Include="$(PackageRID)" Condition="'$(BuildOnUnknownPlatforms)' != 'false'">
+    <_buildingOnRID Include="$(OutputRid)" Condition="'$(BuildOnUnknownPlatforms)' != 'false'">
```

Comment:
Why should we change this? I thought the buildingRid should match the host machine, no?

Reply:
Good catch! I missed the:

```xml
<!-- During NuGet restore, grab assets for all applicable RIDs. -->
```

comment. Reverted. :)